hack/render-sync.sh

```diff
@@ -42,4 +42,5 @@ rendersync manual-cluster/performance base/performance default
 rendersync bootstrap-cluster/performance pinned-cluster/default bootstrap/no-mcp
 rendersync bootstrap-cluster/performance pinned-cluster/default bootstrap-cluster/extra-mcp bootstrap/extra-mcp
 rendersync --owner-ref none -- base/performance manual-cluster/performance no-ref 
-rendersync --owner-ref none -- base/performance manual-cluster/cpuFrequency default/cpuFrequency 
\ No newline at end of file
+rendersync --owner-ref none -- base/performance manual-cluster/cpuFrequency default/cpuFrequency 
+rendersync --owner-ref none -- base/performance manual-cluster/arm default/arm
```

Comment:
Nit: Every text file should really end in EOL character in my view (and apparently the tooling's view too). Let's fix this.